app/assets/stylesheets/base.scss

```diff
@@ -144,3 +144,10 @@ body > footer p {
     display: block;
   }
 }
+
+.notice {
+  background-color: #5d202657;
```

Comment:
Which color is this?

Reply:
A variation of a dark red website's theme color.